Components/LayoutProperties/index.js

```diff
@@ -27,28 +31,29 @@ export default function LayoutProperties({ settings, onSettingsChange }) {
 
   return (
     <>
-      <StyledH3>Layout</StyledH3>
-      <form onSubmit={handleSubmit}>
-        <GridProperties
-          settings={settings}
-          onSettingsChange={onSettingsChange}
-        />
-        <RangeProperties
-          settings={settings}
-          onSettingsChange={onSettingsChange}
-        />
-        <LogScaleProperties
-          settings={settings}
-          onSettingsChange={onSettingsChange}
-        />
-        <Container $centered="center">
-          <InputTypeSubmit
-            value="Finish"
-            onClick={handleLayoutProperties}
-            disabled={notCompletedLayoutProperties()}
-          />
-        </Container>
-      </form>
+      {hasCompletedDistProp === true && (
```

Comment:
Also here: You could simplify the conditional check `hasCompletedDistProp  === true` by simply inserting `hasCompletedDistProp` directly into the condition, or?

Reply:
Done.